### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by manuel-hoelzl**
Fixed some minor typos, added some articles and reworded an expression.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,24 +5,24 @@
 This project is a game that has been developed in order to help teach kids python.
 The students need to code the behavior of the car to achieve the best score.
 
-Here is video of a race (running code from students):
+Here is a video of a race (running code from students):
```
Comment (nirs) [inline README.md]: There is another PR for this, lets drop it from your changes.

File: README.md [hunk 2]
```diff
@@ -5,24 +5,24 @@
...
 
 <a href="http://www.youtube.com/watch?feature=player_embedded&v=BEV-CcqTOnw
 " target="_blank"><img src="rose-video-preview.jpg"
 alt="ROSE Race Car Game" width="860" height="720" border="10" /></a>
 
 In this game, two race cars compete to achieve the most points.
-The race car have to recognize the race track, the obstacles, and the bonus areas;
-calculate the best path to take to avoid the pitfalls; and collect bonus points.
+The race cars have to recognize the race track, the obstacles and the bonus areas.
+Calculate the best path where the pitfalls are avoided and all the bonus points are collected.
```
Comment (nirs) [inline README.md]: I don't think this improves the text, lets drop it from this PR.

File: README.md [hunk 3]
```diff
@@ -5,24 +5,24 @@
...
 The cars move autonomously on the screen within the race track game with no interference
 from the students. No joystick or mouse would be used.
  
-In order to control the car movements, the students needs to implement a 'driver'.
+In order to control the car movements, the students need to implement a 'driver'.
```
Comment (nirs) [inline README.md]: Looks good.

File: README.md [hunk 4]
```diff
@@ -5,24 +5,24 @@
...
 This code is controlling the car and will decide what will be the next action of the car.
 
 For each type of obstacles there is a different action, and different points.
 
-See [examples/README](examples/README) for explanation on how to write a driver module.
+See [examples/README](examples/README) for an explanation on how to write a driver module.
```
Comment (nirs) [inline README.md]: Looks good

File: README.md [hunk 5]
```diff
@@ -31,7 +31,7 @@ To install the dependencies, run:
 
     pipenv install
 
-Or in the old-fashioned way
+Or in the old-fashioned way:
```
Comment (nirs) [inline README.md]: Looks good

File: README.md [hunk 6]
```diff
@@ -58,16 +58,16 @@ Command line interface
 
 You can control the game from the command line using the rose-admin tool.
 
-To start a race, use rose-admin tool on any machine:
+To start a race, use the rose-admin tool on any machine:
```
Comment (nirs) [inline README.md]: Looks good

File: README.md [hunk 7]
```diff
@@ -58,16 +58,16 @@ Command line interface
...
 
     ./rose-admin <server-address> start
 
-To stop a race, use rose-admin tool on any machine:
+To stop a race, use the rose-admin tool on any machine:
```
Comment (nirs) [inline README.md]: Looks good

File: README.md [hunk 8]
```diff
@@ -58,16 +58,16 @@ Command line interface
...
 
     ./rose-admin <server-address> stop
 
-To modify the game rate, you can use set-rate command. The following command
-would change game rate to 10 frames per second:
+To modify the game rate, you can use the set-rate command. The following command
+would change the game rate to 10 frames per second:
```
Comment (nirs) [inline README.md]: Looks good

**Comment by nirs** [comment]
@rollandf, we need manual rebase, merging master into the patch will not help.

We need to:
```
git checkout -b manuel-hoelzl-patch-1 master
git pull https://github.com/manuel-hoelzl/ROSE.git patch-1
git rebase -i master
```
Then fix the conflicts, remove the changes we don't want to take, and leave the changes we want.

Finally merge locally into master, and push to github.

**Comment by rollandf** [comment]
Thanks @manuel-hoelzl  !!

**Pull request merged**